### DISCUSSION TITLE
[help] custom output files in tar_quarto_rep()

### DISCUSSION THREAD

**Discussion by joelnitta** [ropensci/targets#1376] [Help] [answered]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

Is there a way to provide custom names for the files generated with `tar_quarto_rep()`? In particular, I'd like the output file names to include the parameters used when rendering, instead of the hash.

**Comment by wlandau** [comment, marked as answer]
You can include an `output_file` column in the `execute_params` data frame: https://docs.ropensci.org/tarchetypes/reference/tar_quarto_rep.html#arg-execute-params

**Reply by joelnitta** [reply]
Thanks! As usual you've already provided the answer in the package. Sorry I missed it in the docs.
